protoc-gen-pbandk/lib/src/jvmTest/kotlin/CodeGeneratorTest.kt

```diff
@@ -76,11 +73,96 @@ class CodeGeneratorTest {
         assertFalse("enum should not be nullable") { mainClazz.memberProperties.find { it.name == "enum" }!!.returnType.isMarkedNullable }
     }
 
-    private fun compileProto(inputProto: String): KotlinCompilation.Result {
+    @Test
+    fun testKotlinPackageMappingSimple() {
+        val result = compileProto("simple.proto", "kotlin_package_mapping=foobar->newname.pkg")
+
+        assertEquals(ExitCode.OK, result.exitCode, result.messages)
+
+        // Ensure classes and fields were generated successfully
+        result.classLoader.loadClass("newname.pkg.Message1").kotlin
+        result.classLoader.loadClass("newname.pkg.Message2").kotlin
+
+        assertTrue(kotlin.runCatching { result.classLoader.loadClass("foobar.Message1").kotlin }.isFailure)
```

Comment:
Use `assertFails { ... }` instead of `assertTrue(runCatching { ... }.isFailure)`. It's simpler and will produce a more useful error if the test were to fail.

Reply:
I did not know about `assertFails`, I used `assertThrows` in the past but couldn't find that method. Let me update 😄